tools/validators/ontology_validator/yamlformat/validator/unit_lib.py

```diff
@@ -28,31 +29,49 @@
 STANDARD_UNIT_TAG = 'STANDARD'
 
 
+_MeasurementAlias = NamedTuple('MeasurementAlias', [
+  ('alias_name', str),
+  ('base_name', str),
+  ('file_context', findings_lib.FileContext)
+])
+
+
 class UnitUniverse(findings_lib.FindingsUniverse):
   """Helper class to represent the defined universe of units."""
 
   def _GetNamespaceMapValue(self, namespace):
     """Helper method for FindingsUniverse._MakeNamespaceMap.
 
-    Used to create a map from namespace names to unit maps.
+    Used to create a map from namespace names to namespaces.
 
     Args:
       namespace: UnitNamespace
 
     Returns:
-      The units map from the namespace.
+      The namespace.
     """
-    return namespace.units
+    return namespace
```

Comment:
i'm not seeing where this function is being used, but it seems pretty pointless now? can we just delete this?

Reply:
This is part of the way FindingsUniverse works and it's populating the _namespace_map that the other methods use.

Reply:
ok, makes sense to leave it for now